app/assets/javascripts/angular/directives/graph_chart.js

```diff
@@ -134,24 +276,24 @@ angular.module("Prometheus.directives").directive('graphChart', ["$location", "W
             switch (maybeCond) {
               case ">":
                 if (axesBounds[matchingAxis.id].max > maybeYMax) {
-                  graphMax = axesBounds[matchingAxis.id].max;
-                  bound.max = axesBounds[matchingAxis.id].max;
-                } else {
-                  bound.max = maybeYMax;
-                  graphMax = maybeYMax;
-                }
-                scaleID = s.axisID;
-                break;
+                graphMax = axesBounds[matchingAxis.id].max;
+                bound.max = axesBounds[matchingAxis.id].max;
+              } else {
+                bound.max = maybeYMax;
+                graphMax = maybeYMax;
+              }
+              scaleID = s.axisID;
+              break;
               case "<":
                 if (axesBounds[matchingAxis.id].max < maybeYMax) {
-                  graphMax = axesBounds[matchingAxis.id].max;
-                  bound.max = axesBounds[matchingAxis.id].max;
-                } else {
-                  bound.max = maybeYMax;
-                  graphMax = maybeYMax;
-                }
-                scaleID = s.axisID;
-                break;
+                graphMax = axesBounds[matchingAxis.id].max;
+                bound.max = axesBounds[matchingAxis.id].max;
+              } else {
+                bound.max = maybeYMax;
+                graphMax = maybeYMax;
+              }
+              scaleID = s.axisID;
+              break;
```

Comment:
ah sheet, yeah this is completely unrelated, my editor must have picked this up and messed it up.